src/main/java/dev/andrylat/vomelianchuk/card_number_operations/LuhnAlgorithm.java

```diff
@@ -0,0 +1,72 @@
+package dev.andrylat.vomelianchuk.card_number_operations;
+
+import java.util.List;
+
+
+
+public class LuhnAlgorithm {
+    private static final String ERROR_CANNOT_DETERMINE_PAYMENT_SYSTEM = "Payment System can't be determine";
```

Comment:
Not clear: what is relation between control sum and payment system?

Reply:
Fixed here: 7a87509